__tests__/requests.test.js

```diff
@@ -108,6 +107,36 @@ describe("getReviews", () => {
   });
 });
 
+describe("postComment", () => {
```

Comment:
Great start, but there are some extra tests we want to be adding here :
Status 201, ignores unnecessary properties
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist

Reply:
Thanks. What is meant by 'ignores unnecessary properties?

Reply:
If an addtional property other than author or body is included in the sent comment body then the post should not be impacted.